framework/src/Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared/Pages/Shared/Components/AbpPageSearchBox/AbpPageSearchBoxViewComponent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Volo.Abp.AspNetCore.Mvc.UI.Theme.Shared.Pages.Shared.Components.AbpPageSearchBox
+{
+    public class AbpPageSearchBoxViewComponent : AbpViewComponent
+    {
+        public async Task<IViewComponentResult> InvokeAsync()
+        {
+            return await Task.FromResult(View("~/Pages/Shared/Components/AbpPageSearchBox/Default.cshtml"));
```

Comment:
No need to make async if you simply return the view:

`public IViewComponentResult Invoke()`

Reply:
I tried to make invoke method as return type  IViewComponentResult  but I got error like this
![taskReturn](https://user-images.githubusercontent.com/37196959/83390235-29672b00-a3fa-11ea-9055-3b46d5cb8a97.PNG)

thats why I create this method as async method

Reply:
`public IViewComponentResult Invoke()` should work (exactly like that, not `public IViewComponentResult InvokeAsync()`)

Reply:
I fixed it, you can check it